packages/cli/src/commands/test.ts

```diff
@@ -42,8 +42,16 @@ async function storeTest(tmpDir: string) {
 }
 
 export default class Test extends Command {
+  static args = [
+    {
+      name: 'path',
+      description: 'The path where the FastStore being tested is. Defaults to cwd.',
+    }
```

Comment:
question: isn't it necessary to add `required: false`?

Reply:
For this version of oclif/core the `required` is optional, which makes me think that if it's not there it will be assumed as `false`. I've checked this assumption and it holds.

Reply:
This is the type definition: https://github.com/oclif/core/blob/v1.16.4/src/interfaces/parser.ts#L6-L15